tests/mocha/procedures_test.js

```diff
@@ -505,7 +484,7 @@ suite('Procedures', function() {
           this.workspaceSvg.clearUndo();
           Blockly.Events.setGroup('g1');
           this.barDef.setEnabled(false);
-          Blockly.Events.setGroup(false);
+          // Blockly.Events.setGroup(false);
```

Comment:
Can this be removed?

Reply:
I commented this temporarily as i was debugging why a certain warning was happening.